ci/prow/check-commits-number

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
I'd rename this to `ci/prow/check-commits-count`. WDYT?

Reply:
Fixed.